cogs/customchannels.py

```diff
@@ -23,6 +23,8 @@ def __init__(self, bot, utils):
         self.offtopic_category = self.utils.settings['custom_channels']['category']
         self.public_channels = self.utils.settings['custom_channels']['public_channels']
         self.private_channels = self.utils.settings['custom_channels']['private_channels']
+        self.invite_message = "{member.mention} is toegevoegd aan het kanaal!"
```

Comment:
As this would be user-specified, add it to the settings as well, just like the rest.